src/protocol/codec/declarations.c

```diff
@@ -234,6 +235,7 @@ z_result_t _z_decl_commons_decode(_z_zbuf_t *zbf, uint8_t header, bool *has_exte
         _z_zint_t len;
         _Z_RETURN_IF_ERR(_z_zsize_decode(&len, zbf));
         if (_z_zbuf_len(zbf) < len) {
+            _Z_INFO("Not enough bytes to read");
```

Comment:
Seems we need to add _Z_LOG_LVL_WARN (not in this PR, but will be nice to have it for cases like it)

Reply:
Yes that's what I was thinking about. Will open an issue